<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewdatabase</class>
 <widget class="QMainWindow" name="viewdatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>751</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="filterComboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Film</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TV Show</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="searchBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="columnSelector">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Entry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Table View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>115</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Entry by ID number for Deletion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
